openmdao/utils/om.py

```diff
@@ -3,8 +3,11 @@
 """
 
 import sys
+import re
```

Comment:
was this left over?  does not seem to be needed

Reply:
Yep, that was left over. Fixed